src/main.ts

```diff
@@ -1 +1,61 @@
-console.log('Hello Feeders!');
+import 'reflect-metadata';
+import { ElectronReloader } from './reloader';
+import { resolve as pathResolve } from 'path';
+import { app as electronApp, BrowserWindow } from 'electron';
+import { config } from './config';
+
```

Comment:
Volta com o hello feeders! Gosto dele! [exemplo de RC]
```suggestion
import 'reflect-metadata';
import { ElectronReloader } from './reloader';
import { resolve as pathResolve } from 'path';
import { app as electronApp, BrowserWindow } from 'electron';
import { config } from './config';

console.log('Hello Feeders!');
```

Reply:
Não! [exemplo resposta]